doc/wintabs.txt

```diff
@@ -200,6 +200,32 @@ default: ' '
                 '[]' and ')(' are some of the good ones. It's recommended that
                 they have the same length as separators.
 
+*g:wintabs_ui_show_vimtab_name*
+values: 0, 1 or 2
+default: 0
+                This controls how Vim tabs will be displayed.
+                0: show a number of the tab.
+                1: show a name of the tab.
+                2: show number and name of the tab. (<number>:<tab_name>).
```

Comment:
I think your use case is fair. I'm OK with this option, but please remove the format option `g:wintabs_ui_vimtab_separator` as I don't think it's good to implement it that way. A good way to make this feature would be to allow user to free form the tab name like `"%d: %n"`, but that's probably too much for this change.